docs/admin/runai-setup/cluster-setup/cluster-install.md

```diff
@@ -38,7 +38,7 @@ The table provides the following columns:
 * **Status**&mdash;the status of the cluster. For more information see [Cluster status](#cluster-status). Hover over the information icon to see a short description and links to troubleshooting.
 * **Creation time**&mdash;the timestamp the cluster was created.
 * **URL**&mdash;the URL that was given to the cluster at the time of creation.
-* **Run:ai cluster version**&mdash;the version of the cluster.
+* **Run:ai cluster version**&mdash;the Run:ai version installed on the cluster.
 * **Run:ai cluster UUI**&mdash;the unique ID of the cluster.
 
 ### Cluster Status
```

Comment:
@jasonnovichRunAI I think the new sections should be after "Verify your installation"